docs/quickstart.md

```diff
@@ -84,13 +84,14 @@ export OPSTRACE_NAME=<choose_a_name>
 The name will globally identify you in our domain as `$OPSTRACE_NAME.opstrace.io`, which we provide for you by default as a convenience.
 
 Then, you'll create a simple [configuration file](./references/cluster-configuration.md) with the most basic options.
-Note that we define a tenant named `myteam` to send our application metrics to, which is separate from the `system` tenant that hosts internal metrics.
+Note that we define two tenants named `staging` and `prod` that are separate from the `system` tenant that hosts internal metrics; we will later send our application metrics to `staging` to demonstrate how data can be isolated between logical units such as deployment environments.
```

Comment:
- Could specify `internal metrics about Opstrace`
- May as well be two sentences (replace ; with .); it's pretty long as a single sentence
- `We will send our application` (skip `later`)